src/Symfony/Bundle/FrameworkBundle/Tests/Console/Descriptor/TextDescriptorTest.php

```diff
@@ -15,6 +15,11 @@
 
 class TextDescriptorTest extends AbstractDescriptorTest
 {
+    protected function setUp()
+    {
+        putenv('COLUMNS=121');
```

Comment:
Done, thanks. The same should apply to the 3.2 branch and `SymfonyStyleTest`. See #21456